### ISSUE TITLE
Is falun gong worse than other relisions, and what should the government do about it?

### ISSUE THREAD

**Issue by cirosantilli** [cirosantilli/chat#3]
My understanding:

## Is Falun Gong worse than other religions?

To my understanding, same reasons as any other religion, e.g. Christianity, Buddhism.

The only difference is that Falun Gong is recent, and all the power is concentrated by the Master. So if he is evil or crazy, that is serious.

But I haven't seen any such evidence so far.

## What should the government do about it?

Unless someone convinces me it is much worse than other religions, the government should do nothing about it.

People want to do it, and I don't think they are harming others, so we must let them do it.

If we start forbidding things that 5% of the population need very much, very soon we reach a point in which everyone is harmed.

Tolerance is the best path.

---

Please feel free to give me arguments that show otherwise. I just want the truth. Cheers.

**Comment by rppig** [comment]
Here’s my counter argument part one.

First of all, I’m  an agnostic, but I’m root for the freedom of religion.

But why should people believe in a religion? Because the belivers think that the claims of those religions are true or right.

As an agnostic, I don’t know whether Buddha or Jesus is divine because they are from the ancient. Even though the possibility of their divine story is extremely low, it still can happen. We'll never know the truth until we invent the time machine. :)

But for Falun Gong, we can prove that what Falun gong said is false by confronting with the Master Li.

Let’s say one day I claim that I have invented a awesome chip that can cure most of the diseases by embedding the chip into human body. Of course it’s  a lie. But I sell a lot and make a lot of money by using some PS modified pictures and hiring some actors to act as the paients . 

What should the government do about me?

Falun Gong is doing the same thing, except that I’m using high technology while Falun Gong is using beliefs.

So unlike the other religions from the ancient, Falun Gong can be proved to be false. Master Li claims that he can be floating on air, then do it on TV live instead of showing pictures, so do the other so called  miracles. If he can’t, then the government should do something to protect the people. If the government lets it alone, people are easily blind faith especial for those who lack of education. Unluckily, the majority of Chinese are lack of basic education because of the rediculos ‘Culture Revolution’ in 20th century.

Different country has different policies. I’m not a fan of our country’s administration and I really hate the Great Fire Wall, but I’m root for the ban of Falun Gong.

As my personal experience, Falun Gong was very popular when I was a child. A great many people including my grandparents believe in it. They believed that it can cure all the diseases only if they could follow Master Li and they should do anything Li told them to do. So people would not go to hospital even when they were sick. It looks like a apocalyptic that almost everybody is crazy.

## Short version ##

We cannot confront with Jesus,

we cannot confront with Buddha,

We can confront with Li Hongzhi.

**Comment by cirosantilli** [comment]
@rppig thanks for the reply!

I reiterate my question: what **exactly** should the government do about people who do Falun Gong in your opinion? E.g.:

- make anti falun gong publicity (done today, similar to anti-smoking campaings)
- make them pay fines
- put them in jail (apparently what is done today?)
- kill them
- something else

Equally important: what kind of trial should Falun Gong people have:

- no trial, straight to jail (apparently what is done today?)
- public trial like any other crime

This point is crucial because if forbidding something may have costs (humanitarian, etc.) much greater than allowing it. These are of course hard considerations made today about drugs and prostitution in the west. 

## I extract two points main from what you said

1. if something is false and people believe in it, we should punish them
2. people don't go do hospital because of Falun Gong

please correct me if I got them wrong or missed other important ones.

## if something is false and people believe in it, we should punish them

I disagree with this point.

E.g., in the past, people though the earth was flat, and Galileo almost went to prison for saying otherwise.

But the real problem is for political questions. E.g.: I say: hey government, democracy is best for all of us! Govt says: false, I have made many deep considerations and you are wrong. Therefore, go to jail.

Furthermore, we cannot prove that Li is a fake. He can always dodge any physical test with: "only certain people can see it", or "I can't do it in front of non-believers". Those are not satisfactory to convince me of truth of course, but also don't allow me prove its false.

## people don't go do hospital because of Falun Gong

If this point is true, I agree that it is very serious.

I'm not convinced about the factual accuracy of this fact, in particular if its serious enough to pay the price of making it illegal.

Let's move all discussion to http://skeptics.stackexchange.com/questions/27529/is-it-true-that-falun-gong-practicioners-dont-go-to-hospital-when-they-need-it if it does not get closed there (the more broad http://skeptics.stackexchange.com/questions/27431/is-falun-gong-worse-for-its-practitioners-and-society-than-other-religions-like did get closed, so lets see).

---

BTW I'm also agnostic and I don't believe that Li, Jesus or Buddha were magical.

**Comment by mookrs** [comment]
My English is poor, just say something I know.

I’m an agnostic, too. In my opinion, if lots of people do Falun Gong, I support to educate the public. Putting them into jail is totally wrong, however, China is one-party system, the government did what were rough but powerful.

I can't express well what is "educate the public", but if you've seen a U.S. TV series *The Newsroom*, perhaps you could know my means.

I don't think @rppig expressed "If something is false and bad for society and someone says it is true, we should punish them." (@rppig, do you?) He said Falun Gong was evil, Master Li was a liar. He didn't say punishing believer is right. Even he thinks so, I don't. The government put believer into jail just for maintaining its reign. 

"we can never decide what is true or false." Yes, we can. The earth is round, this is true. For political questions, what is better or worse can be decided. The government chose a better choice for itself, in some respects, the choice has benefit for public. (Of course it is rough.)

Falsifiability is a complicated conception. I judge it by science, common sense and reason. If I can't, I doubt it.

"people don't go do hospital because of Falun Gong" It has been a long time for me since I knew someone believed Falun Gong. I was also a child at that time. But I believe this exists. I find some posts on Zhihu [1][2] (a site in China like Quora), although they are not focus on talking Falun Gong. But if you can read Chinese, see the answers and comments (quite a lot), you may find something useful. (Because of Internet censorship and humor, it may be even more difficult to understand what they actually say...)

Personally speaking, I don't like Falun Gong, it lies as the government. Most of the believer have low education. I always think Falun Gong is ridiculous.

Apologize for my English level, hope someone else can give a much more excellent reply. Have a good day!

[1] http://www.zhihu.com/question/23988890
[2] http://www.zhihu.com/question/27659795

**Comment by cirosantilli** [comment]
@mookrs thanks for the reply! Your points are valid and your English was good. I'll take a look at the links. And I'm glad that your mind is open to allowing others do something, even if you think it is ridiculous.

For curiosity: do you think FLG is more ridiculous than the other major religions?

> Most of the believer have low education.

Do you have links to studies on that one? You may very well be right, I'd just like to have more references. Anecdotally my GF tells me that there are many well educated practitioners as well in the US, but of course, we need statistics to decide.  Related discussion at: http://skeptics.stackexchange.com/questions/27274/do-non-religious-people-make-more-money

**Comment by cirosantilli** [comment]
Teaser (not very serious point): consider that Falun Gong practitioners directly perceive "energies" mentioned by the master, just like you and I perceive light. Similarly, some people perceive some political choices as good, and others bad.

**Comment by mookrs** [comment]
Falun Gong has different parts. It came from Qigong (气功), something like Yoga. It really can help us having a good body. Nothing wrong in this part, I think in western countries or Taiwan or Hong Kong, they emphasis this part.

But if you look at its theory part... Totally ridiculous. The theory mix buddhism and taoism (a Chinese origin religion), Master Li's tone just like a businessman. He said there was a wheel in our belly... I know it's hard for you to read (Traditional) Chinese, but someday you can have a look at [《轉法輪》](http://big5.falundafa.org/chibig5/zfl.htm). His odd theory is like a brochure for selling goods...

So, I think Falun Gong is more ridiculous than the other major religions I know.

> Do you have links to studies on that one? 

I'm afraid not. There are few studies on Falun Gong in China, at least I don't know the education level in the country. My opinion comes from my relatives. I know three believers when I was a child living in a county town: a man as taxi driver, a woman as a junior high school teacher, and a woman as a farmer. They are all about forty. The second link in my previous reply show a father technical secondary school graduate education (中专) and two old men who believed it. If they have high education, they may believe in science, just like you and me.

I find something interesting to support "people don't go do hospital because of Falun Gong". In [《法輪功》](http://gb.falundafa.org/chigb/flg.htm) [Chapter 5](http://gb.falundafa.org/chigb/flg_5.htm), Master Li says "practicing Falun Gong but still taking medicine means you don't believe practicing can heal disease". This is a hint.

> 弟子：炼功还用不用吃药？
> 师：这个问题自己悟，炼功吃药就是不相信炼功能治病，信你还吃什么药。可是你不按心性标准要求自己，出了问题你会说李洪志不叫吃药，可是李洪志还叫你严格要求你的心性，你做到了吗？真正修炼大法的人，身上带的都不是常人的东西，常人得的病都不允许在你身上得。你的心如果摆正的话，相信炼功能炼好，把药停了，不去管，不去治，就有人给你治了。大家在这里一天比一天好，一天比一天舒服，是怎么回事？好多人身上有我的法身不断的進出，忙忙活活的，就是在帮你做这些事。如果自己心里不稳定，一边炼功，一边采取不相信或试试看的态度，那你什么也得不到。你相不相信佛，是由你的悟性、根基所决定的。如果佛显现出来，就在这里可以用肉眼看的清清楚楚，那人们都去学佛，就不存在思想转换问题。你要先相信，然后才能看的到。

**Comment by mookrs** [comment]
> there are many well educated practitioners as well in the US

Many means how many? This is hard to say. In the other hand, I think well educated practitioners treat Falun Gong as something like Qigong or Yoga. This can improve your body, I believe it. But I don't think Falun Gong will tell them "practicing Falun Gong but still taking medicine means you don't believe practicing can heal disease", they are more like a social group, not like a religion in western countries.

**Comment by cirosantilli** [comment]
I think you are right about the emphasis difference. My GF tells me the stuff they believe in, I see why you think it is ridiculous. But I still can't disprove it.

But I think it is equally ridiculous as believing that people have lived to 930 years old (Adam), coming back from the dead (Jesus, Buddhism in some form), a man can open the seas (Moises), flying magical human beings with wings (angels), talking donkeys who fight for animal rights (bible https://www.biblegateway.com/passage/?search=Numbers+22%3A21-39), heaven, hell, God, or any other "non scientific" belief.

> Many means how many

As I've mentioned it is purely anecdotal (== I have no hard evidence), you may be right.

**Comment by mookrs** [comment]
@cirosantilli Most stories in religions are ridiculous. Because I'm sure about this, I become an agnostic... Falun Gong stole many conception from other religions, for this reason, I don't respect it like other religion. User "筝弦 Cochan" said these in <http://www.zhihu.com/question/23988890/answer/26309820>'s comments:

> (法轮功) 没有自己原创教义，从拜弥勒教抄点，从白莲教抄点，配上完全不沾边的气功，生造出一个所谓宗教出来

So you konw it's hard for me to respect Falun Gong.

**Comment by cirosantilli** [comment]
@mookrs I agree that it has taken stuff from multiple other religions, and added a few new ones.

But I don't understand why that reduces its respect to you:

- if it used concepts from other religions, and you respect the other religions, then shouldn't you have even more respect for it, as it is more likely to have reused concepts from other religions that you respect?
- I bet all religions take things which the guy that made them up heard at the time and put together. 

    For Christians, I am sure half of what they have comes from the Jews + stuff jesus made up + stuff the church made up.

    And Jewish beliefs come from several even older cults: https://en.wikipedia.org/wiki/Ancient_Semitic_religion which were even polytheist (multiple gods, a big no-no for Jews)

**Comment by mookrs** [comment]
This is not a 1 + 1 = 2 question. It's like programming. I like creator, I don't like copycat. I like Linux which from Unix, I don't like someone gives me a system just change Linux GUI skin, and even says "I create it, this is the best system."

Another example, if I like tea and milk, it doesn't mean I have to like milk tea.

Back to religion, there is also no culture in Falun Gong. It's just Master Li and his team's theory. If you know more about Chinese, you will find the theory's style of writing is suck.

**Comment by cirosantilli** [comment]
> This is not a 1 + 1 = 2 question. It's like programming. I like creator, I don't like copycat. I like Linux which from Unix, I don't like someone gives me a system just changes Linux GUI skin, and even says "I create it, this is the best system."

I didn't say you have to like it because it is a mixture, only that it is like every other religion: all of them are mixtures, and all of them say they're the best.

> It's just Master Li and his team's theory

Same for all religions.

>  you will find the theory's style of writing is suck.

I will take your word for :-)

**Comment by cirosantilli** [comment]
Ah, also found now that the old testament and quran say you should kill non believers: 

- http://www.quora.com/Does-Christianity-ever-advocate-or-condone-killing-non-believers 
- http://www.quora.com/Where-in-the-Quran-does-it-say-that-non-believers-must-be-killed

That's kind of worse than not taking medicine I think :-)

**Comment by mookrs** [comment]
"It's **just** Master Li and his team's theory." means there is no culture in Falun Gong. Only theory, no culture.

Mixture problem I don't want to explain any more, if you get my point, It's better; if not, also OK.

**Comment by mookrs** [comment]
Sigh... Totally wrong key point... But the two links can't change the fact: there were people who didn't take medicine, and maybe some of them died for this reason.

如果在一本 20 世纪末出版的宗教教义中加入唆使人谋杀他人的条目，这无疑是犯罪。宣扬或暗示有病不治的理念，已经是这个时代相当邪恶的教义了。（Forgive me I used Chinese）

This is my last reply, hope others can explain your other questions. :smile:

**Comment by cirosantilli** [comment]
@mookrs I didn't say that the medicine point is not true, only that I don't think it's worse than other religions. ~~

Thanks for all links and info! Cheers ~~

**Comment by rppig** [comment]
Hi @cirosantilli ,

A few days ago, I wrote a lot to dispute your points and then suddenly, I realized that it's useless for arguing. So I deleted them all.

There are such many beautiful and nice things in China, meanwhile there are also lots of weaknesses of our administration and people. It's very hard to judge others' behaviours but at least I can make myself a better man so the world will be a little better too.

Have a nice day~~

**Comment by cirosantilli** [comment]
@rppig 

> A few days ago, I wrote a lot to dispute your points and then suddenly, I realized that it's useless for arguing. So I deleted them all.

Please don't! :-) Arguing is the basis of democracy! I respect your opinions very much and I really want to know what you think. In many points, I'm not really sure about the truth.

> There are such many beautiful and nice things in China, meanwhile there are also lots of weaknesses of our administration and people. It's very hard to judge others' behaviours but at least I can make myself a better man so the world will be a little better too.

Agree with all.

All best rppig!

**Comment by YongHaoWu** [comment]
@cirosantilli  
Hi, I had seen all the arguing in this issue. :p
I think many people had explained clearly to you about the FaLunGong while you still can not have the same opinion.
As far as I am concerned, the root cause is that you support completely democracy and personal freedom.
Let me put this straight, if there was a religion asked all the people believed in it to commit suicide, through which they can enjoy the heaven while suffer in the hell.
You don't think this should be banned because the people trusting in this religion and committing suicide is their will.
Right? :P
Sry for my poor English.

**Comment by cirosantilli** [comment]
@YongHaoWu thanks for dropping in, your english is not poor.

> Let me put this straight, if there was a religion asked all the people believed in it to commit suicide, through which they can enjoy the heaven while suffer in the hell.

Yes, this is a tough question to me...

The right to suicide alone without the religious component is already an interesting debate which will never reach a conclusion: https://en.wikipedia.org/wiki/Right_to_die

Personally, I am more extreme towards personal freedom than most people. I think people should do whatever they want in life as long as it does not hurt others. If they want to kill themselves, they should do it. I think there should places that let people kill themselves painlessly for money if they want to die (but yes, those places should require an inscription beforehand, and a wait of 6 months before they kill you in case you change your mind).

Then there is also the question of what "hurt others" mean... yes if you kill yourself, it will hurt your parents a lot. I don't have a general answer to that problem...

I also think that it it too hard to prevent people from not doing something like killing themselves. If they want to do it, they will do it.

Yes, when it comes to a religion, things get more complicated, as they might order people to do things they would not do without the religion... but then again, I don't see how to prevent it. If people want to do it, they will do it.

Of course, suicide is a taboo in a sin in most religions, and such a religion wouldn't last for long because most people would be dead (dark humour :-) )

Also such a religion would likely not attract many people, since most people don't like suicide.

Summary:

- I would allow such a religion, but I wouldn't like it
- I would be less likely to put them in my profile name  :-) 

That said, if I were a normal "average" person that would forbid such religion, I think I would still support FLG as much as other major religions like Christianity, since to me they are on the same level of craziness.

**Comment by YongHaoWu** [comment]
Bingo.
I think the completely freedom is the difference between you and other people dicussed with you(include me). Maybe most Chinese are in favor of their opinion instead of completely personal freedom because of the culture background for thousands of years. Thus, there are no such a religion like Falungong will hurt themselves and people in China.

Though I love the idea of completely freedom, for average person, I think FaLunGong should be banned because people that don't think or lack of knowledge will believe in it easily. As people lack of rational consideration and easily ruled by madness,  Government should exist and control this with many people's thinking and discussion. For example, banning of drugs.

Ps: The resources of FaLunGong in wikipedia is doubtful.

**Comment by cirosantilli** [comment]
@YongHaoWu  You think FLG is beyond what should be allowed. I disagree, because I think it is like all other religions, and the cost of forbidding is too high. That is the hole point of discussing how bad it is.

Also consider the costs of forbidding: things: if drugs were legal, there would be no organized crime. For that reason, light drugs like cigarettes and alcohol are not banned. Same for FLG: when you put someone in jail, they cannot work and make society richer.

What source about FLG is not doubtful?
